setup.py

```diff
@@ -71,6 +71,7 @@
         "html2text>=2016.1.8, <=2016.9.19",
         "hyperframe>=4.0.1, <5",
         "jsbeautifier>=1.6.3, <1.7",
+        "kaitaistruct>=0.5",
```

Comment:
We want an upper bound here to guard against upgrade in kaitaistruct that break compatibility. Can we make this `>=0.5, <0.6` please?